Orange/widgets/tests/base.py

```diff
@@ -561,3 +566,61 @@ def _compare_selected_annotated_domains(self, selected, annotated):
         selected_vars = selected.domain.variables + selected.domain.metas
         annotated_vars = annotated.domain.variables + annotated.domain.metas
         self.assertLess(set(selected_vars), set(annotated_vars))
+
+
+class datasets:
+    @staticmethod
+    def path(filename):
+        dirname = os.path.join(os.path.dirname(__file__), "datasets")
+        return os.path.join(dirname, filename)
+
+    @classmethod
+    def data_1(cls):
```

Comment:
So maybe just missing_data_n (with matching/modified filenames of the test files)?

I am afraid that otherwise the pattern will continue and we will get data_{4,5,6,7,8} in the future, making it impossible to tell what kind of data is used in the test.